akka-testkit/src/test/scala/akka/testkit/AbstractSpec.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.testkit
+
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.Matchers
+import org.scalatest.WordSpecLike
+
+// we could migrate AkkaSpec to extend this
+abstract class AbstractSpec extends WordSpecLike with Matchers with BeforeAndAfterEach
```

Comment:
I'm not sure this is worth it - indeed it perhaps removes some duplication, but replaces it by indirection...

Reply:
@raboof Yes, at present it is not worth it. But my thinking here is in future it would be - in a separate commit, perhaps next sprint, I'd like to migrate any other tests not needing to extend AkkaSpec to this or similar, and remove scattering usage of `org.scalatest` around. That way if things change in future we have less time needed on maintenance and fewer changes.